benchmark/test_select_and_slice_perf.py

```diff
@@ -4,18 +4,15 @@
 import torch
 
 from .attri_util import FLOAT_DTYPES
-from .performance_utils import GenericBenchmark2DOnly, generate_tensor_input
+from .performance_utils import GenericBenchmark, generate_tensor_input
 
 
-class TensorSelectBenchmark(GenericBenchmark2DOnly):
+class TensorSelectBenchmark(GenericBenchmark):
+    def set_more_metrics(self):
+        return ["gbps"]
+
     def set_more_shapes(self):
```

Comment:
What for if it only calls base class's method with the same name?

Reply:
There's something I don't like about object oriented design. But here let's consider set_more_metrics is called on  TensorSelectBenchmark.